metadata/configuration/concepts/1230_child_hiv_exposed_initial_form.csv

```diff
@@ -4,14 +4,21 @@ uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:
 246eafad-87a8-4eac-afa6-dc06e7cd74c0,,,Pathologie,Pathology,Pathologie,Pathology,,Misc,N/A,,
 a2e7c8c5-e1cb-4f06-abb3-25d92d491ac8,,Normal;Requires monitoring;Pathology,Développement psychomoteur (Entourer),Psychomotor development,Développement psychomoteur (Entourer),Psychomotor development,,Misc,Coded,,
 c40513be-48e2-4e3c-aced-31e8659efa45,,,Conclusion d'examne clinique,Clinical Examination Conclusion,Conclusion d'examne clinique,Clinical Examination Conclusion,,Misc,Text,,
+9644653a-b9af-11ed-afa1-0242ac120002,,Yes full name;No full name,CTX à 6 semaine,CTX at 6 weeks,CTX à 6 semaine,CTX at 6 weeks,,Misc,Coded,,
 7cf24fae-6d68-411c-8306-4d72b31cd7fe,,Yes full name;No full name,Prophylaxies (Cotrimoxazole),Prophylaxis (Cotrimoxazole),Prophylaxies (Cotrimoxazole),Prophylaxis (Cotrimoxazole),,Misc,Coded,,
 b575fb76-4c99-4603-8455-8801850fef80,,,NVP Enfante,NVP child,NVP Enfante,NVP child,,Misc,N/A,,
 c2d1934c-ccce-4ed2-9669-d0f383807c51,,,TARV meternelle,TARV maternal,TARV meternelle,TARV maternal,,Misc,N/A,,
 f30e7233-18ab-4b16-87ae-a442f0acd1f4,,Normal;NVP child;TARV maternal,Traitement ARV post partum ,Post partum ARV treatment,Traitement ARV post partum ,Post partum ARV treatment,,Misc,Coded,,
 57baae85-5dc9-4d6c-bee7-5528021a31e2,,,À jour selon l'âge,Up to date for age,À jour selon l'âge,Up to date for age,,Misc,N/A,,
 fff0aeac-c1c4-4ddc-8ac1-ccad8618d6cd,,,Pasa jour,Overdue,Pasa jour,Overdue,,Misc,N/A,,
 270eb876-11ba-454a-aaa7-5a9adaa4eb2e,,Up to date for age ;Overdue,Etat vaccinal,Immunization status,Etat vaccinal,Immunization status,,Misc,Coded,,
-fe7f4bcc-7334-4361-b88d-c610b6586a4a,Psychomotor development;Clinical Examination Conclusion;Prophylaxis (Cotrimoxazole);Post partum ARV treatment;Immunization status,,3. Consultation,3. Consultation,3. Consultation,3. Consultation,,Misc,N/A,,
+ce17a311-4492-4ec7-b36f-6395cdc32350,,,sevré après l’allaitement,discharged after breastfeeding,sevré après l’allaitement,discharged after breastfeeding,,Misc,N/A,,
+0907567c-5e85-423b-af07-817fe1724324,,,Négatif(ve) encore sous allaitement,Negative still breastfeeding,Négatif(ve) encore sous allaitement,Negative still breastfeeding,,Misc,N/A,,
+e4d7d750-e03e-45d7-9021-04f8670c51e8,,,PDV( Perdu de vue),LTFU,PDV( Perdu de vue),LTFU,,Misc,N/A,,
```

Comment:
@tresorluz there is a concept `Lost to follow up` (fr: `Suivi arrêté`) with uuid `9e5c23f7-b545-41f9-8ff3-5de2dd2e8730`, it's also of class Misc and dataype N/A. Let's confirm with JP if the existing concept can be re-used here?